sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/InvocationResolvesToOverrideWithParams.cs

```diff
@@ -62,10 +62,11 @@ protected override void Initialize(SonarAnalysisContext context)
 
         private static void CheckCall(SyntaxNode node, ArgumentListSyntax argumentList, SyntaxNodeAnalysisContext context)
         {
-            if (!(context.SemanticModel.GetSymbolInfo(node).Symbol is IMethodSymbol invokedMethodSymbol) ||
+            if (argumentList == null ||
+                argumentList.Arguments.Count == 0 ||
+                !(context.SemanticModel.GetSymbolInfo(node).Symbol is IMethodSymbol invokedMethodSymbol) ||
                 !invokedMethodSymbol.Parameters.Any() ||
-                !invokedMethodSymbol.Parameters.Last().IsParams ||
-                argumentList == null)
+                !invokedMethodSymbol.Parameters.Last().IsParams)
```

Comment:
We can include in this "if" the condition from next one to remove one extra exit point from this method.